app/src/main/java/com/framgia/localchat/SettingsActivity.java

```diff
@@ -0,0 +1,59 @@
+package com.framgia.localchat;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+
+public class SettingsActivity extends AppCompatActivity {
+
+    private LinearLayout mLinearRowContainer;
+    private Menu mMenu;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_settings);
+        mLinearRowContainer = (LinearLayout) findViewById(R.id.linear_row_container);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+    }
+
+    private void inflateRow(int position) {
+        LayoutInflater inflater = (LayoutInflater) getSystemService(LAYOUT_INFLATER_SERVICE);
+        final View rowView = inflater.inflate(R.layout.settings_row, null);
+        final EditText etName = (EditText) rowView.findViewById(R.id.et_name);
```

Comment:
please initialize all the widget eg.(textview,button ) in a particular method  and also make a eventListenerManager() to set listener.